src/ui/components/ProtocolTimeline.tsx

```diff
@@ -55,27 +78,24 @@ export default function ProtocolTimeline() {
   }
 
   return (
-    <div className="absolute -top-4 flex w-full flex-col space-y-1">
-      <Spans regions={loadedRegions.loading} color="bg-gray-500" title="Loading" />
-      <Spans regions={loadedRegions.loaded} color="bg-orange-500" title="Loaded" />
-      <Spans regions={loadedRegions.indexed} color="bg-green-500" title="Indexed" />
+    <div className="absolute -top-8 flex w-full flex-col space-y-1">
+      {/* This is pretty silly, but these are the classnames that get generated
+      dynamically by this component. And if they are not statically in the
+      build, then tailwind will not include the rules for them, which makes them
+      not work.  */}
```

Comment:
YUP definitely ran into this before (which is why gray/orange/green are hardcoded :P)